src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/VolleyUnitOfWork.cs

```diff
@@ -5,10 +5,12 @@
     using Context;
     using Contracts;
 
-    /// <summary>
-    /// Defines Entity Framework implementation of the IUnitOfWork contract.
-    /// </summary>
+#pragma warning disable S3881 // "IDisposable" should be implemented correctly
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)

Reply:
We agreed that this suppress will be removed

Reply:
ok, deleted that